src/Horse.Provider.Console.pas

```diff
@@ -78,10 +78,10 @@   THorseProvider<T: class> = class(THorseProviderAbstract<T>)
     class property IOHandleSSL: THorseProviderIOHandleSSL read GetIOHandleSSL write SetIOHandleSSL;
     class procedure Listen; overload; override;
     class procedure StopListen; override;
-    class procedure Listen(APort: Integer; const AHost: string = '0.0.0.0'; ACallback: TProc<T> = nil); reintroduce; overload; static;
-    class procedure Listen(APort: Integer; ACallback: TProc<T>); reintroduce; overload; static;
-    class procedure Listen(AHost: string; const ACallback: TProc<T> = nil); reintroduce; overload; static;
-    class procedure Listen(ACallback: TProc<T>); reintroduce; overload; static;
+    class procedure Listen(APort: Integer; const AHost: string = '0.0.0.0'; ACallback: TProc<T> = nil; ACallbackStopListen: TProc<T> = nil); reintroduce; overload; static;
```

Comment:
You can rename param ACallback to ACallbackListen?

Reply:
will be done